client/src/lib/utils/date.ts

```diff
@@ -42,3 +42,71 @@ export const makeDayRow = (date: Date) => {
     .forEach((day: number) => daysArray.push(day));
   return daysArray;
 };
+
+export const getRangeFromDate = (date: Date) => {
+  const { year, month, day } = getYMD(date);
+  return {
+    from: new Date(year, month, day - FRONT_HALF + 1),
+    to: new Date(year, month, day + REAR_HALF),
+  };
+};
+
+/**
+ *
+ * @param target date 보다 앞에있는지 확인할 date 객체
+ * @param date 확인의 기준이 되는 date 객체
+ */
+export const isFormer = (target: Date, date: Date): boolean => {
+  const targetYMD = getYMD(target);
+  const dateYMD = getYMD(date);
+  return (
+    targetYMD.year < dateYMD.year ||
+    targetYMD.month < dateYMD.month ||
+    (targetYMD.month === dateYMD.month && targetYMD.day < dateYMD.day)
+  );
```

Comment:
```suggestion
  const dateOption = {year:"2-digit", month:"2-digit", day:"2-digit"}
  return target.toLocaleString("ko-KR", dateOption) < date.toLocaleString("ko-KR", dateOption)
```
으로 바꿀 수 있을 것 같습니다. 😉
`// output : '21. 06. 18.'`

Reply:
와우...처음 보네요. JS 고수시군요